src/QirTools/pyqir/qirlib/src/emit/intrinsics.rs

```diff
@@ -0,0 +1,95 @@
+use inkwell::module::Module;
+use inkwell::values::FunctionValue;
+
+pub const M: &str = "M";
+pub const R_X: &str = "Rx";
```

Comment:
Can some of the duplicated definitions here possibly be reduced by a `macro_rules!` macro?